pkg/env/env.go

```diff
@@ -346,7 +347,7 @@ func (e *testEnv) Run(m *testing.M) int {
 			if e.cfg.DisableGracefulTeardown() {
 				panic(rErr)
 			}
-			klog.Error("Recovering from panic and running finish actions", rErr)
+			klog.Errorf("Recovering from panic and running finish actions", rErr, string(debug.Stack()))
```

Comment:
If you are using `klog.Errorf` you would have to use the `%` format specifiers. Otherwise it will just get appended without space. 

```suggestion
			klog.ErrorS(rErr, "Recovering from panic and running finish actions", "stack", string(debug.Stack()))
```
might be a better way to setup the logger ?

Reply:
`klog.ErrorS(rErr, "Recovering from panic and running finish actions", "stack", string(debug.Stack()))` did not work because rErr is not of error type. 
I fixed previous by adding % format specifiers